e2e/cypress/support/commands.ts

```diff
@@ -3,6 +3,16 @@ import { eventExist } from '../support/helpers';
 // Cypress Commands: any action that could be taken in any test
 // any command added in here must also have its signature added to index.d.ts
 
+
```

Comment:
This is the global functions file where all the functions in the tests were moved to. All the cypress tests can use the functions in this file.